frontend/src/core/resource/components/ResourceMenu.js

```diff
@@ -113,22 +150,70 @@ export class ResourceMenuBase extends React.Component<Props, State> {
                         autoFocus
                         value={ this.state.search }
                         onChange={ this.updateResourceList }
+                        placeholder="Filter resources"
+                    />
+                </div>
+
+                <div className="resources-list-header">
+                    <Localized id="resource-ResourceMenu--resource">
+                        <span className="resource"
+                            onClick={ this.sortByResource }>Resource</span>
```

Comment:
We nest element attributes like this:
```html
<span
    className="resource"
    onClick={ this.sortByResource }
>
    Resource
</span>
```

Please also do that for the remaining `<span>` elements below.